### DISCUSSION TITLE
Flat button component

### DISCUSSION THREAD

**Discussion by buenojosias** [tasteui/tasteui#528] [General]
Previously I used another UI package that, among the buttons, had a button style called flat button. It was similar to the outline button (with transparency and hover effect, but without border).
Is it possible to do this in my project using TallStackUI personalization? Or is it possible to add this feature in future package updates?

**Comment by devajmeireles** [comment]
I think it's possible to do this through soft customization, but I think it's better to have it as a new default option for the buttons. I'm getting ready to work on some TSUI stuff next week, I'll add that to my to-do list.

**Reply by buenojosias** [reply]
> I think it's possible to do this through soft customization, but I think it's better to have it as a new default option for the buttons. I'm getting ready to work on some TSUI stuff next week, I'll add that to my to-do list.

Perfect! I will wait anxiously.

**Reply by devajmeireles** [reply]
Done! Released at v1.32: https://github.com/tallstackui/tallstackui/releases/tag/v1.32.0
